src/asm3/dbupdate.py

```diff
@@ -242,6 +243,26 @@ def fclob(name, nullable = True):
     sql += index("adoption_ReturnedReasonID", "adoption", "ReturnedReasonID")
     sql += index("adoption_ReturnedByOwnerID", "adoption", "ReturnedByOwnerID")
     sql += index("adoption_TrialEndDate", "adoption", "TrialEndDate")
+    sql += index("adoption_AdoptionEventId", "adoption", "AdoptionEventID")
+
+    sql += table("adoptionevent", (
```

Comment:
I think when we originally discussed this, I mentioned that adoption events sounded like a superset of #940 which was generic fundraising events.

I'd prefer more generic names - eg: just call the tables event and eventanimals

event could also have a type field so that adoption event could be set as the type (just a fixed list of integers no need for a lookup). Initially though I just add it as a column and not do anything with it since adoption events will be first.

This leaves it open for extending for other types of events and uses in future, such as the ability place an eventid  column on a payment (ownerdonation table) to track money received as part of an event, etc.